.github/workflows/visual_testing.yml

```diff
@@ -0,0 +1,25 @@
+name: Visual testing
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  chromatic:
+    name: Visual testing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Chromatic
+        run: npx chromatic --project-token=91e12320417c
```

Comment:
I know this token looks scary. But this is the public project token, it's safe to commit this in code :)

Reply:
Chromatic dev here. Normally this could go in a secret, but those aren't available on the `pull_request` trigger on public repositories, so hardcoding the token is the recommended approach. This token does not provide access to any sensitive data. It can only be used to create new Chromatic builds, and we have mechanisms in place to deal with abuse.

Since you are running this in CI and the project is linked to GitHub, you can pass `--exit-once-uploaded` to speed up the CI job.